migration-from-froala.md

```diff
@@ -97,7 +97,7 @@ The following examples show an initial Froala configuration and the migrated Tin
 </html>
 ```
 
-#### TinyMCE 5 - A Basic Configuration
+#### TinyMCE {{site.productmajorversion}} - A Basic Configuration
```

Comment:
Should this be here, since I thought this would break links?

Reply:
fixed